docs/docsite/rst/network_aggregate_resources.rst

```diff
@@ -0,0 +1,175 @@
+*********************************
+Aggregate Resources in Networking
+*********************************
+
+.. contents:: Topics
+
+This section explores how and when `Aggregate Resources` can be used within Ansible Networking.
+
+This document is part of a collection on Networking. The complete list of guides can be found at :ref:`Network Guides <guide_networking>`.
+
+Overview
+========
+
+Consider the case if you wanted to ensure a set of vlans are present on a switch. Although at a high level this sounds like a simple request, there in a slight nuance:
+
+* Should these vlans be in *addition* to the existing configuration?  - *additive*
+* Should these vlans be the *only* ones present? - *aggregate*
+
+
+.. versionadded:: 2.4
+
+   The ``aggregate:`` option has been added in Ansible 2.4 and is available in certain modules, see the modules documentation to see if the feature is available.
+
+Additive resources
+===================
+
+Continuing our vlan example, the following task ensure that vlans ``1``, ``2`` and ``3`` are in the states specified in the task, in `addition` to any existing vlans.
+
+This task *will not* change any vlans already configured on the switch, apart from the ones specified in the task. Ansible will ensure that the vlans specified in the task exist with the `name` and `state` specified.
+
+.. code-block:: yaml
+
+   - name: configure vlans neighbor in addition to existing
+     net_vlan:
+       vlan_id: “{{ item.vlan_id }}”
+       name: “{{ item.name }}”
+       state: “{{ item.state | default(‘active’) }}”
+     with_items:
+       - { vlan_id: 1, name: default }
+       - { vlan_id: 2, name: mgmt }
+       - { vlan_id: 3, state: suspend }
+
+FIXME Should we even document ``with_items``, is that just confusing the matter. Should this be the same as `Aggregate resources` though without ``purge: yes``?
+
+FIXME Only showing ``with_items`` to show how it could be done
+
+
+Would be better written as
```

Comment:
Consider changing to something like "the preceding example would be better written as follows:"